src/store/reducer.js

```diff
@@ -1,6 +0,0 @@
-import { combineReducers } from 'redux';
-import entitiesReducer from './entities';
-
-export default combineReducers({
-  entities: entitiesReducer,
-});
```

Comment:
Keeping entity objects inside entities makes the project more organized.

Removing this code would break other team member's code, kindly stick to previously agreed code structure.